cloud build.yaml

```diff
@@ -0,0 +1,21 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args: [
+    '-c',
+    'docker pull us.gcr.io/$PROJECT_ID/$REPO_NAME:latest || true',
+  ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [
+            'build',
+            '--pull',
+            '--target', 'application',
+            '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:latest',
+            '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+            '--cache-from', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:latest',
+            '.'
+        ]
+images: [
+  'us.gcr.io/$PROJECT_ID/$REPO_NAME:latest',
```

Comment:
Looks like this will create "latest" tag even for non-master builds.
Here's how we can tag it only on master: https://github.com/getsentry/sentry-release-registry/blob/master/cloudbuild.yaml#L19-L25